src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/AsciidocToHtml.kt

```diff
@@ -0,0 +1,55 @@
+package nl.avisi.structurizr.site.generatr.site.model
+
+import org.asciidoctor.Asciidoctor
+import org.asciidoctor.Options
+import org.asciidoctor.SafeMode
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import nl.avisi.structurizr.site.generatr.site.asUrlToFile
+
+fun asciidocToHtml(pageViewModel: PageViewModel, asciidoc: String, svgFactory: (key: String, url: String) -> String?): String {
+    val asciidoctor = Asciidoctor.Factory.create()
+    val options = Options.builder()
+            .safe(SafeMode.SERVER)
+            // Docs dir needs to be exposed from structurizr, which is not the case at the moment.
+            // Needed for partial include `include::partial.adoc[]`, which structurizr also does not support.
+            // see https://docs.asciidoctor.org/asciidoc/latest/directives/include/
+            // another option could be https://docs.asciidoctor.org/asciidoctorj/latest/locating-files/#globdirectorywalker-class
+            // .baseDir(File("./docs/example/workspace-docs"))
```

Comment:
Could you please expand that comment with the consequences of not having a `baseDir`?